document/FileFormat.mdk

```diff
@@ -758,7 +786,7 @@ by means of the `result`'s `ruleId` property (&sect;[#result-ruleId]) or `ruleKe
 
 ### `properties` property { #run-properties }
 
-A `run` object may contain a property named `properties`, whose value is property bag.
+A `run` object may contain a property named `properties` whose value is property bag.
```

Comment:
fixed

---
In reply to: [75180912](https://github.com/sarif-standard/sarif-spec/pull/256#discussion_r75180912) [](ancestors = 75180912)